test/unit/plugins/rest_spec.js

```diff
@@ -266,42 +405,41 @@ describe("The Rest plugin", function () {
 				});
 			});
 		});
-
 	});
 
 	describe("when there is a problem with the database connection", function () {
-			var mapper = new MemoryMapper();
-			var server = new Hapi.Server();
-
-			before(function () {
-				Sinon.stub(mapper, "create").rejects(new Error("Simulated Failure."));
-				server.connection();
-				return server.registerAsync({
-					register : Rest,
-					options  : {
-						mapper : mapper
-					}
-				});
-			});
+		var mapper = new MemoryMapper();
```

Comment:
I don't understand what changed in these test cases. This lines look unrelated to the SSH functionality.